docs/src/docs/reports/prioritymessages.adoc

```diff
@@ -0,0 +1,44 @@
+= Priority Messages
+
+== For test methods
+
+In case you would like to have some log messages displayed on a prominent place on the test details page, you can log priority messages like this:
+
+[source,java]
+----
+    Logger log() {
```

Comment:
This is not wrong but Testerra provides the interface `Loggable` for easy access to `log()`:

Please extend the code snippet with 

````
public class MyTestClass extends TesterraTest implements Loggable {
   ...
}
````

Reply:
ok.

Reply:
... and please remove the the log() method in line 11 ;-)